packages/paste-website/src/pages/experiences/console/index.mdx

```diff
@@ -0,0 +1,379 @@
+- Update the search JSON and create a PR in the repository
+
+## Copy
+
+All Console copy should be presented in sentence case. For in-depth copy guidelines,
+see [Paste's Content Guidelines](/foundations/content/content-checklist).
+
+### Naming
+
+Partner with the Console team for support whenever you are creating navigation and menu names in Console to
+ensure they are maintaining a consistent experience and are meaningful to users.
+
+Naming should ideally be 22 characters or less to fit on a single line and cannot exceed 44 characters (2 lines).
+
+The H1 title of a page should correlate with the name in the sidebar navigation, although you can use a
+shortened version of the name in the sidebar navigation to save space, when necessary. Navigation items
+should be succinct and descriptive. Avoid using commonly shared terms like “Programmable” in the navigation
+as it inhibits scannability. It is ok, for example, to say “Messaging” in the navigation but
+“Programmable Messaging” in the H1 of the page.
+
+## Console product pre-launch checklist
+
+- Request a short product description for the Product Explorer page card (35-60 characters) by filing a jira in WEBREQ project and tagging Lauren Kaye
+- Review all copy in the nav and within your product at Content office hours
+- Review your designs with the Design Systems team for an accessibility check and ensure proper use of components, design language, and patterns.
+- Review your navigation structure with the Console team at Console team office hours (link)
+- File Jira with Marketing (MARK) to create new product logo
+  - Once it is ready, you need to add your product icon to Paste
+- Add key search terms for your pages to the search database
+- Your product's error events are being published to the error logs
+
+## Regional
+
+When a product appears in more than one region, the following changes should be made to the design:
+
+- On the product explorer page, the product card should have a dropdown caret next to the pin allowing the user to pin the product to the desired region.
+
+![](../../../assets/images/experiences/console/regional-1.png)
+
+<ImageCaption>Example of a product with a pinned region</ImageCaption>
+<br />
+<br />
+
+- On the product explorer page, the product card should have a “Regional” badge and popover that signifies this product is available in more than one region.
+
+![](../../../assets/images/experiences/console/regional-2.png)
+
+<ImageCaption>Example of popover explaining a product is regional</ImageCaption>
+
+- On the Side navigation, when an account has products pinned in more than one region, region headers, such as “United Statues (US1)” will appear above those products in both “Develop” and “Monitor”.
+
+![](../../../assets/images/experiences/console/regional-3.png)
+
+<ImageCaption>Example of region headers in both Develop and Monitor</ImageCaption>
+<br />
```

Comment:
What would you think of removing these extra break in between each of the bullet points in this section? IMO the margin below each image caption creates enough separation between each bullet without the additional breaks.

Reply:
Yeaaahh you right you right. Extra breaks have been _eliminated_